rfcs/rfc-involuntary-dissolutions.md

```diff
@@ -0,0 +1,210 @@
+
+- Start Date: 2024-04-26
+- Target Major Version:
+- Reference Issues: n/a
+- Entity Issue: [20937](https://github.com/bcgov/entity/issues/20937)
+- Implementation PR: (leave this empty)
+
+
+## Overview
+
+The involuntary dissolution process will consist of two BE scheduled jobs - involuntary dissolutions job and furnishings job.
+
+This process will be mostly automated with staff input only in the form of updating a configuration.  The configuration determines how many new businesses can be added to the dissolution process on a per day basis.
+
+The involuntary dissolutions job will manage the overall dissolution process for batches of businesses where the dissolution process is kicked off.
```

Comment:
So the batch can contain more businesses than the configuration controls? In that case, what happens?

Reply:
No, that's not possible.  

When the voluntary dissolution job runs, it will grab the value for **NUM_DISSOLUTIONS_ALLOWED** from the **configurations** table to determine how many new businesses it can initiate the involuntary dissolution process for.  

Remaining eligible businesses will be picked up in subsequent job runs.